hpx/util/itt_notify.hpp

```diff
@@ -117,12 +122,14 @@ HPX_EXPORT void itt_thread_set_name(char const*);
 HPX_EXPORT void itt_thread_ignore();
 
 HPX_EXPORT void itt_task_begin(___itt_domain const*, ___itt_string_handle*);
+HPX_EXPORT void itt_task_begin(___itt_domain const*, ___itt_id*,
+    ___itt_string_handle*);
 HPX_EXPORT void itt_task_end(___itt_domain const*);
 
 HPX_EXPORT ___itt_domain* itt_domain_create(char const*);
-HPX_EXPORT ___itt_string_handle* itt_task_handle_create(char const*);
+HPX_EXPORT ___itt_string_handle* itt_string_handle_create(char const*);
```

Comment:
I don't even know what version is the correct one... But using the newest VTune is probably a safe bet ;)